conans/client/generators/msbuild.py

```diff
@@ -0,0 +1,208 @@
+import os
+import textwrap
+from xml.dom import minidom
+
+from conans.client.tools import msvs_toolset, VALID_LIB_EXTENSIONS
+from conans.errors import ConanException
+from conans.model import Generator
+from conans.util.files import load
+
+
+class MSBuildGenerator(Generator):
+
+    dep_props = textwrap.dedent("""\
+        <?xml version="1.0" encoding="utf-8"?>
+        <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+            <ImportGroup Label="PropertySheets" >
+            </ImportGroup>
+            <PropertyGroup Label="UserMacros" />
+            <PropertyGroup />
+            <ItemDefinitionGroup />
+            <ItemGroup />
+        </Project>
+        """)
+
+    dep_conf_props = textwrap.dedent("""\
+        <?xml version="1.0" encoding="utf-8"?>
+        <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+          <ImportGroup Label="PropertySheets">
+            {transitive_imports}
```

Comment:
```suggestion
        {transitive_imports}
```

Just to fix indent issues